.github/workflows/ci.generate.ts

```diff
@@ -343,6 +361,13 @@ const ci = {
             job: "test",
             profile: "release",
             skip_pr: true,
+          }, {
+            os: Runners.macosArm,
+            job: "test",
+            profile: "release",
+            // TODO(mmastrac): We don't want to run this M1 runner on every main commit because of the expense.
+            skip:
+              "${{ github.event_name == 'pull_request' || github.ref == 'refs/heads/main' }}",
```

Comment:
Nice! In the future, perhaps we could have this also run infrequently on main.

Reply:
We might be able to do something silly like "if commit ID ends in zero, run aarch64 build"